src/java/edu/stanford/muse/util/Util.java

```diff
@@ -1132,7 +1132,7 @@ public static String commatize(long n)
 	/** if num > 1, pluralizes the desc. will also commatize the num if needed. */
 	public static String pluralize(int x, String desc)
 	{
-		return Util.commatize(x) + " " + desc + ((x != 1) ? "s" : ""); // want plural even if x is 0, e.g. "0 messages"
+		return Util.commatize(x) + " " + desc + ((x != 1) ? "s " : ""); // want plural even if x is 0, e.g. "0 messages"
```

Comment:
why add space after s? that is not natural and symmetrical with the case where there is no s.